### PR TITLE
feat(FOROME-1080): add disabling button and input while saving

### PR DESCRIPTION
**Pull request by QSdmitrioul**
[Task](https://quantori.atlassian.net/browse/FOROME-1080)

### CODE REVIEW

**Review by AlMaQntr — APPROVED** [review]

File: src/pages/filter/dtree/components/modals/components/modal-save-dataset.tsx [hunk 1]
```diff
@@ -38,6 +38,8 @@ export const ModalSaveDataset = observer(() => {
   const pathName = history.location.pathname
   const { wsList } = mainTableStore
   const isDone = operations.savingStatus[1] === 'Done'
+  const disabled =
```
Comment (AndreyUstyumenko) [inline src/pages/filter/dtree/components/modals/components/modal-save-dataset.tsx]: isSaving

Comment (AndreyUstyumenko) [inline src/pages/filter/dtree/components/modals/components/modal-save-dataset.tsx]: isSaving

Comment (QSdmitrioul) [reply src/pages/filter/dtree/components/modals/components/modal-save-dataset.tsx]: Fixed

Comment (QSdmitrioul) [reply src/pages/filter/dtree/components/modals/components/modal-save-dataset.tsx]: Fixed

**Review by AndreyUstyumenko — APPROVED** [review]

**Pull request merged**